mesh/mesh-io-tcpserver.c

```diff
@@ -388,35 +416,53 @@ static bool tls_ctx_init(struct mesh_io_private *pvt)
 	return true;
 }
 
-static bool process_tls_args(char *argv[],
-				size_t argc, struct mesh_io_private *pvt)
+static bool acl_entry_added(struct tcpserver_acl *acl, uint8_t *identity_buf,
+								void *user_data)
 {
-	size_t len;
+	char			identity[IDENTITY_LEN];
+	struct mesh_io		*mesh_io = user_data;
+	struct tcpserver_conn	*conn = tcpserver_conn_new(identity_buf);
 
-	pvt->uuid = l_util_from_hexstring(argv[1], &len);
+	if (!l_queue_push_tail(mesh_io->pvt->conns, conn))
+		return false;
 
-	if (!pvt->uuid || len != UUID_LEN) {
-		l_error("Invalid UUID length");
+	if (!hex2str(identity_buf, IDENTITY_BUF_LEN, identity,
+							     sizeof(identity)))
```

Comment:
I'm somewhat confused if `tcpserver_conn` holds binary or textual representation of peer's identity.

I think we should keep the binary representation in memory and convert strings incoming from TLS handshake only to find a corresponding PSK.

Reply:
`tcpserver_conn` keeps binary representation of peer's identity. I've found that in rest of code, binary representation is called with `_buf` postfix, so I've decided to call it similar.